src/estimation/msm_criterion.py

```diff
@@ -91,7 +95,20 @@ def _build_and_evaluate_msm_func(
             end_date=spring_end_date,
             debug=debug,
             group_share_known_case_path=share_known_path,
+            initial_states_path=initial_states_path,
         )
+    elif mode == "combined":
+        res_spring = _build_and_evaluate_msm_func_one_season(
+            params=params,
+            seed=seed + 84587,
+            prefix=prefix,
+            start_date=spring_start_date,
+            end_date=spring_end_date,
+            debug=debug,
+            group_share_known_case_path=share_known_path,
+            initial_states_path=None,
```

Comment:
Not sure if None is correct here. @janosg How do you make sure here that the simulation is continued?